<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeersDialog</class>
 <widget class="QDialog" name="PeersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="ClassicPeersTab">
    <attribute name="title">
     <string>Classic</string>
    </attribute>
    <widget class="QTreeView" name="peersTreeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>681</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="BuyerPeersTab">
    <attribute name="title">
     <string>Buyers</string>
    </attribute>
    <widget class="QTreeView" name="buyersTreeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>681</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SellerPeersTab">
    <attribute name="title">
     <string>Sellers</string>
    </attribute>
    <widget class="QTreeView" name="sellersTreeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>681</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ObserverPeerTab">
    <attribute name="title">
     <string>Observers</string>
    </attribute>
    <widget class="QTreeView" name="observersTreeView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
